src/containers/ImagesContainer.jsx

```diff
@@ -8,7 +8,10 @@ import Image from 'components/Image/Image.jsx';
 
 class ImagesContainer extends Component {
   componentDidMount() {
-    this.props.loadImages();
+    const { images } = this.props;
+    if (!images || !images.length) {
+      this.props.loadImages();
```

Comment:
if images are not loaded, fetch it, but since this is loaded from before hand we don't need